tests/test_cli.py

```diff
@@ -364,3 +346,76 @@ def my_test_function(a, b):
                 ret = cli.main(['apply', 'my_test_module'], stdout, stderr)
     assert ret == 0
     assert 'warning:' not in stdout.getvalue()
+
+
+def test_apply_stub_using_libcst():
+    source = """
```

Comment:
nit: can we use `textwrap.dedent()` to avoid having all this code at top-level indent?